docs/dev/topology-over-raft.md

```diff
@@ -154,8 +154,20 @@ transition. An example of this is the `rebuilding` state which does not change
 the topology but requires streaming data.
 
 Separately from the per-node requests, there is also a 'global' request field
```

Comment:
The commit descritption is too vague, it needs to describe the change without assumming the context of the PR:

> 
> docs: updated topology-over-raft.md

docs: topology-over-raft: Document truncate_table request

> 
> This change updates topology-over-raft.md by adding the description of
> the new topology request.

Redundant if you use mine suggestion, because the headline says it all.